azure-container-apps/ReadMe.md

```diff
@@ -1,337 +1,346 @@
 5. Select **Save**.
 6. Select **Create** to deploy a new revision using the updating image.
 7. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).
 
-## Resource recommendations
+## Appendix: Resource recommendations
 
-The 1Password SCIM Bridge Pod should be vertically scaled when provisioning a large number of users or groups. Our default resource specifications and recommended configurations for provisioning at scale are listed in the below table:
+The pod for 1Password SCIM Bridge should be vertically scaled you provision a large number of users or groups. These are our default resource specifications and recommended configurations for provisioning at scale:
 
 | Volume    | Number of users | CPU   | memory |
 | --------- | --------------- | ----- | ------ |
 | Default   | <1,000          | 0.25  | 0.5Gi  |
 | High      | 1,000–5,000     | 0.5   | 1.0Gi  |
 | Very high | >5,000          | 1.0   | 1.0Gi  |
 
-If provisioning more than 1,000 users, the resources assigned to the SCIM bridge container should be updated as recommended in the above table. The resources specified for the Redis container do not need to be adjusted.
+If you're provisioning more than 1,000 users, update the resources assigned to the SCIM bridge container to follow these recommendations. The resources specified for the Redis container do not need to be adjusted.
 
-#### Default
+### Default
 
-To revert to the default specification that is suitable for provisioning up to 1,000 users:
+To revert to the default specification that is suitable for provisioning up to 1,000 users, run the following command:
 
 ```sh
 az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
   --cpu 0.25 --memory 0.5Gi
 ```
 
-#### High volume deployment
+### High volume deployment
 
-For provisioning up to 5,000 users:
+If you're provisioning up to 5,000 users:
 
 ```sh
 az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
   --cpu 0.5 --memory 1.0Gi
 ```
 
-#### Very high volume
+### Very high volume
 
-For provisioning more than 5,000 users:
+If you're provisioning more than 5,000 users:
 
 ```sh
 az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
   --cpu 1.0 --memory 1.0Gi
 ```
 
-## If Google Workspace is your identity provider
-<details>
-<summary>Connect Google Workspace using the Azure Cloud Shell or AZ CLI</summary>
-The following steps only apply if you use Google Workspace as your identity provider. If you use another identity provider, do not perform these steps.
+## Get help
```

Comment:
Another option instead of `Get Help` could be `Common Troubleshooting Tips`.

Reply:
I think "Get help" is ideal since it's consistent with other articles, and some of the headers aren't _technically_ troubleshooting, but rather specs. We could rename the headers though...